study.md

```diff
@@ -21,18 +21,26 @@ There is no requirement to follow any links from these pages.
 After doing all the required readings please tell us anything you were confused about.
 
 ```md
-<!-- your answer here -->
+so personally for my self its much easier to see (either in video or in calss)
+when we are talking about new stuff.
+i got confuse with the function callback mostly after reading the article.
+i feel like i know whats the meaning of comeback but i dont understand how it works.
+
+
 ```
 
 Are there any nagging questions relating to functions that you have?
 
 ```md
-<!-- your answer here -->
+mostly about function declarations and function expressions. i dont really understand when to use what.
```

Comment:
Here's a short bit of code that I wrote to show the differences between function declarations and function expressions.  Let me know if it's unclear.
```javascript
// This will output, "[Function: sayHello]".
console.log(sayHello);

// This will work and output, "Hello".
sayHello();

// This will output, "undefined" since the declaration of "sayHi" has been
// hoisted to the top, but its definition has not been.
console.log(sayHi);

// This will throw an error for the reason above.  Since sayHi is undefined,
// it's trying to execute "undefined" as if it were a function.
sayHi();

// Function declaration.  This gets hoisted to the top.
function sayHello() {
  console.log('Hello.');
}

// Function expression.  This gets hoisted to the top, but its assignment to
// function() { ... } does not.
var sayHi = function() {
  console.log('Hi.');
};

// This will output, "[Function: sayHi]", since it's after the definition of the
// function.
console.log(sayHi);

// This will output, "Hi".
sayHi();
```